dev/merge_arrow_pr.py

```diff
@@ -581,20 +677,20 @@ def cli():
 
     pr.merge()
 
-    if pr.jira_issue is None:
-        print("Minor PR.  No JIRA issue to update.\n")
+    if pr.issue is None:
+        print("Minor PR.  No issue to update.\n")
         return
 
-    cmd.continue_maybe("Would you like to update the associated JIRA?")
-    jira_comment = (
+    cmd.continue_maybe("Would you like to update the associated issue?")
+    issue_comment = (
         "Issue resolved by pull request %s\n[%s/%s]"
```

Comment:
For posting this on GitHub issues, the formatting doesn't need to include the `[` `]` (I would personally also remove the newline and double PR number/url for GitHub (since the url gets rendered as a number anyway)

So something like

"Issue resolved by pull request GH-%s" with just the pr_num (or using the full url, and then the message can maybe be the same for both JIRA and GitHub)

Reply:
Yes, I'm aware of that. I tweaked it a little bit so that it works reasonably on both GitHub and Jira.
See https://github.com/amol-/arrow/issues/3

We can refine it further in the future, at the moment my goal was to make it good enough for both Jira and Github.

PS: I still need to test it against a Jira issue as I did some changes on the Jira side too.